README.rst

```diff
@@ -117,16 +117,11 @@ Installing from Source
 ~~~~~~~~~~~~~~~~~~~~~~
 
 You can create and install a wheel package to install west as well.
-The `wheel`_ Python package is required to do this. See "Installing Wheel"
-below if you need to do this.
 
 To build the west wheel file::
 
-  # macOS, Linux
-  python3 setup.py bdist_wheel
-
-  # Windows
-  py -3 setup.py bdist_wheel
+  pip3 install --upgrade build
```

Comment:
As I said before in https://github.com/zephyrproject-rtos/west/pull/715#discussion_r1757613300, I'm not a fan of using the random version of the day. This is not good for reproducibility.

Reply:
This is done as described by python itself.

Reply:
I think you mean https://packaging.python.org/en/latest/tutorials/packaging-projects/#generating-distribution-archives?

Any author of any package in any language will always tell you to run "the latest" because they're not interested in reports of bugs that they have already fixed. They're only interested in reports of new bugs. That bias is not necessarily aligned with packaging and release goals - notably build reproducibility.

Somewhat related and on-going discussion in #737

Reply:
I built and inspected `west-1.2.99-py3-none-any.whl` on very different Operating Systems: Ubuntu 22, Arch Linux and macOS and I have very good build reproducibility news. Ubuntu 22 shipped with old `python-build` version 0.7.0 while the other two had the latest 1.2.2 version. Yet the content of the wheel packages were identical (except for the expected .zip metadata differences). So `python-build` does not seem to inject any timestamp of version-dependent metadata in the package, which means its version does not seem to affect build reproducibility.